packages/bpk-component-scrollable-calendar/src/BpkScrollableCalendarGridList.tsx

```diff
@@ -75,6 +63,20 @@ const BpkScrollableCalendarGridList = (props: Props) => {
   const endDate = startOfDay(startOfMonth(rest.maxDate));
   const monthsCount = DateUtils.differenceInCalendarMonths(endDate, startDate);
 
+  // These constants are here to facilitate calculating the height
+  // Row and month item heights are defined in rem to support text scaling
+  const ROW_HEIGHT = 2.75;
```

Comment:
Moved this down, so it can be assigned by the optional prop.

Reply:
I think if we need to move any constants within the component and having this run with every render, we need to be careful what we move and also we shouldn't be using capital letters for these

Reply:
https://github.com/Skyscanner/backpack/pull/3057#discussion_r1383539646

Yeah, makes sense.

With us now using updating the ROW_HEIGHT via the prop, we don't need to move all of these constants down.

We will be in a middle state where some dimension properties are still constant whilst others are not and are used within the component.